extractor/src/test/java/org/schabi/newpipe/extractor/services/DefaultStreamExtractorTest.java

```diff
@@ -105,6 +106,12 @@ public void testUploaderVerified() throws Exception {
         assertEquals(expectedUploaderVerified(), extractor().isUploaderVerified());
     }
 
+    @Test
+    @Override
+    public void testSubscriberCount() throws Exception {
+        assertGreaterOrEqual(expectedUploaderSubscriberCountAtLeast(), extractor().getUploaderSubscriberCount());
```

Comment:
... continues (2)
```suggestion
        if (expectedUploaderSubscriverCountAtLeast() == UNKNOWN_SUBSCRIBER_COUNT) {
            assertEquals(UNKNOWN_SUBSCRIBER_COUNT, extractor().getUploaderSubscriberCount());
        } else {
            assertGreaterOrEqual(expectedUploaderSubscriberCountAtLeast(), extractor().getUploaderSubscriberCount());
        }
```
Also, this way if `expectedUploaderSubscriverCountAtLeast()` returns `UNKNOWN_SUBSCRIBER_COUNT`, i.e. its default value, we will make sure the extractor actually returns an unknown subscriber count.